pkg/virtctl/imageupload/imageupload.go

```diff
@@ -207,50 +182,71 @@ func usage() string {
 }
 
 type command struct {
-	clientConfig clientcmd.ClientConfig
+	clientConfig            clientcmd.ClientConfig
+	insecure                bool
+	uploadProxyURL          string
+	name                    string
+	size                    string
+	pvcSize                 string
```

Comment:
nit: Do we need to keep both? We could simply use a local variables during the parsing and here simplify the struct

Reply:
I'll check it. Thanks for pointing about it :)

Reply:
From what I see both have different purposes:
```
cmd.Flags().StringVar(&c.pvcSize, "pvc-size", "", "The size of the PVC to create (ex. 10Gi, 500Mi).")
cmd.Flags().StringVar(&c.size, "size", "", "The size of the DataVolume to create (ex. 10Gi, 500Mi).")
```
And both tested in the unit tests. I'd prefer to keep it as it is.